.gitignore

```diff
@@ -110,3 +110,4 @@ asv.conf.json
 
 # jupyter notebooks
 .ipynb_checkpoints
+.venv
```

Comment:
Please insert new-line 👍🏻

Reply:
ok, done!

Reply:
haha no, a newline _after_ the `.venv` :)

Reply:
ooooh, that makes more sense 😅